### ISSUE TITLE
Add API Gateway costs

### ISSUE THREAD

**Issue by cristim** [sbarski/serverless-calculator#8]

**Comment by jasonpolites** [comment]
There's a PR for this

https://github.com/sbarski/serverless-calculator/pull/15

**Comment by sbarski** [comment]
The PR is now merged. Thank you @jasonpolites. 

I think it's a great start but we can add further features and enable users to model bandwidth costs, etc..

**Comment by jasonpolites** [comment]
Yeah for sure.  Adding network costs would make sense.. but the holy grail is actually an accurate price/performance measure.  Harder to be deterministic about (because there are lots of variables), but a pure cost analysis for compute only really makes sense if the performance is the same, does it scale up/down as fast, do you get more/fewer cold-starts, does latency increase with scale, does a 256MB function on Lambda run faster/slower than a 256MB function on Azure?  Dunno, but it would be interesting to measure/track.
